scout/server/blueprints/variants/controllers.py

```diff
@@ -300,11 +300,10 @@ def parse_variant(
     has_changed = False
     compounds = variant_obj.get("compounds", [])
     if compounds and get_compounds:
-        # Check if we need to add compound information
-        # If it is the first time the case is viewed we fill in some compound information
-        if "not_loaded" not in compounds[0]:
```

Comment:
This change is likely to slow down the loading of the variants page. This is because before update_variant_compounds was called just once if the not_loaded key wasn't present in compound dict. Now it is called every time a variants page is loaded. The thing is you can'y know a priory if the compound is dismissed, or if the dismissed status has changed since last time you were on variantS page. 

We should test it on stage and see how much slower it gets. Any other solution is welcome @dnil

Reply:
Testing is good. Without profiling, it does seem a bit overkill to have the compounds be re-added each time the page is loaded. Perhaps add like a populate_compounds to the controller instead that just quickly fills the relevant data based on existing compound variant ids rather than always going to the case load version? Im fairly sure that one was intended as a one-off.. The dismiss status may have changed, but the gene symbol and compound variant id would still be the same.

If we index the events collection we can go that way instead as well. There will just be a handful of dismiss events for each case, and anyway we know the variant id from the compound already..

Reply:
Yes, I like the second solution better, with the events should be faster! I'll fix this branch accordingly. Brilliant! 👍

Reply:
Good compromise!